README_fr-FR.md

```diff
@@ -91,199 +91,197 @@
-- [![Open-Source Software][OSS Icon]](https://github.com/dbcli/pgcli) [pgcli](https://www.pgcli.com/) - Pgcli is a command line interface for Postgres with auto-completion and syntax highlighting.
-- [![Open-Source Software][OSS Icon]](https://github.com/postgres/postgres) [PostgreSQL](https://www.postgresql.org/download/) - PostgreSQL is a powerful, open source object-relational database system with more than 15 year development. PostgreSQL is not controlled by any corporation or other private entity and the source code is available free of charge
-- [![Open-Source Software][OSS Icon]](https://www.sqlite.org/src/doc/trunk/README.md) [Sqlite](https://sqlite.org/download.html) - SQLite is an in-process library that implements a self-contained, serverless, zero-configuration, transactional SQL database engine.
-- [![Open-Source Software][OSS Icon]](https://github.com/sqlitebrowser/sqlitebrowser) [Sqlite Browser](http://sqlitebrowser.org/) - Visually create, manage, and view sqlite database files.
+- [![Open-Source Software][OSS Icon]](https://github.com/apache/cassandra) [Cassandra](http://cassandra.apache.org/) - La base de données Apache Cassandra est le bon choix lorsque vous avez besoin d'évolutivité et de haute disponibilité sans compromettre les performances. L'évolutivité linéaire et la tolérance aux pannes éprouvée sur le matériel de base ou l'infrastructure cloud en font la plate-forme idéale pour les données critiques.
+- [![Open-Source Software][OSS Icon]](https://github.com/apache?query=couchdb) [CouchDB](http://couchdb.apache.org/) - Synchronisation multi-maîtres transparente, qui évolue du Big Data au mobile, avec une API HTTP / JSON intuitive et conçue pour la fiabilité.
+- [DataGrip](https://www.jetbrains.com/datagrip/) - DataGrip est un IDE multiplateforme qui s'adresse aux administrateurs de base de données et aux développeurs travaillant avec des bases de données SQL. Il dispose de pilotes intégrés qui prennent en charge DB2, Derby, H2, HSQLDB, MySQL, Oracle, PostgreSQL, SQL Server, Sqlite et Sybase.
+- [![Open-Source Software][OSS Icon]](https://github.com/serge-rider/dbeaver/) [DBeaver](http://dbeaver.jkiss.org/) - Un client de base de données universel prenant en charge plusieurs plates-formes et bases de données.
+- [![Open-Source Software][OSS Icon]](https://cgit.kde.org/kexi.git/about/) [Kexi](http://kexi-project.org/) - Kexi est un créateur d'applications de base de données visuelles open source, un concurrent tant attendu pour des programmes comme MS Access ou Filemaker.
+- [![Open-Source Software][OSS Icon]](https://mariadb.org/get-involved/getting-started-for-developers/) [MariaDB](https://mariadb.org/) - L'un des serveurs de bases de données les plus populaires. Fabriqué par les développeurs originaux de MySQL.
```

Comment:
Even tho it is the literal translation for the word "made", the word "fabriqué" sounds inappropriate in that context in my opinion.
In the french meaning of the verb "fabriquer", it makes it kinda sounds like the developers "crafted" this software, like manually with their hands.

I guess the word "conçu" sounds more accurate.
Like so:

`L'un des serveurs de bases de données les plus populaires. Conçu par les développeurs originaux de MySQL.`

This is minor tho :)